src/main/java/frc/robot/RobotContainer.java

```diff
@@ -207,8 +207,11 @@ private void configureButtonBindings() {
     }
 
     if (climbSubsystem != null) {
-      buttons[Constants.AButton].whileHeld(climbUPCommand);
-      buttons[Constants.XButton].whileHeld(climbDOWNCommand);
+      if (Constants.testMode) {
```

Comment:
Carry-over from Controller-Layouts branch, which isn't merged yet/has feedback. Instead of trying to do both at once, could this PR just have the test mode controller layout for climb?

If someone sets testMode = false, we won't have button bindings for climbing. Would be good to do it all at once in Controller-Layouts to account for those cases